framework/src/geomsearch/FindContactPoint.C

```diff
@@ -171,32 +177,74 @@ findContactPoint(PenetrationInfo & p_info,
     }
 
     DenseVector<Real> rhs(dim - 1);
-
     rhs(0) = -dxyz_dxi[0] * d;
 
     if (dim - 1 == 2)
       rhs(1) = -dxyz_deta[0] * d;
 
     DenseVector<Real> update(dim - 1);
-
     jac.lu_solve(rhs, update);
 
-    ref_point(0) += update(0);
+    // Improvised line search in case the update is too large and gets out of the element so bad
+    // that we cannot reinit at the new point
+    Real mult = 1;
+    bool success = false;
+    while (!success)
```

Comment:
```suggestion
    while (true)
```

Reply:
If you don't want to incorporate a suggestion, please give a rationale instead of just marking it as resolved

Reply:
pretty sure I applied the comments lemme look

Reply:
redone. I must have force-pushed it off the branch